build-go.sh

```diff
@@ -134,6 +134,9 @@ function bundleDist() {
 		return $?
 		;;
 	tar.gz)
+		if [ -f "$build_dir/install.sh" ]; then
+			chmod +x $build_dir/install.sh
+		fi
```

Comment:
We leave the executable bit as-set in the upstream repository. Where did you see this bit unset?